firebase-database/src/commonMain/kotlin/dev/gitlive/firebase/database/database.kt

```diff
@@ -78,7 +84,7 @@ internal expect class NativeDatabaseReference : NativeQuery {
     val key: String?
     fun push(): NativeDatabaseReference
     suspend fun setValueEncoded(encodedValue: Any?)
-    suspend fun updateEncodedChildren(encodedUpdate: Any?)
+    suspend fun updateEncodedChildren(encodedUpdate: EncodedObject)
```

Comment:
are these methods suppose to be part of the public API? They don't have an equivalent methods in the official SDK, I'm assuming we don't have any documentation for them either, and personally I have no idea what 'encoded' means here?